mmv1/templates/terraform/examples/spanner_backup_schedule_daily_full.tf.erb

```diff
@@ -0,0 +1,37 @@
+resource "google_spanner_instance" "main" {
+  config       = "regional-europe-west1"
+  display_name = "main-instance"
+  num_nodes    = 1
+}
+
+resource "google_spanner_database" "database" {
+  instance = google_spanner_instance.main.name
+  name     = "<%= ctx[:vars]['database_name'] %>"
+  version_retention_period = "3d"
+  ddl = [
+    "CREATE TABLE t1 (t1 INT64 NOT NULL,) PRIMARY KEY(t1)",
+    "CREATE TABLE t2 (t2 INT64 NOT NULL,) PRIMARY KEY(t2)",
+  ]
+  deletion_protection = false
```

Comment:
Nit: You may want to consider using `true` for docs, so users don't default to turning it off. There are other examples in the repository, but you would use another var to do that.